modules/library/xml/src/test/java/org/geotools/xml/filter/FilterCompliancePreProcessorTest.java

```diff
@@ -0,0 +1,79 @@
+/*
+ *    GeoTools - The Open Source Java GIS Toolkit
+ *    http://geotools.org
+ * 
+ *    (C) 2002-20017, Open Source Geospatial Foundation (OSGeo)
```

Comment:
Also I don't think OSGeo existed that far back (incorporated in 2006), so perhaps just this year...